### PR TITLE
feat: add dev blog tutorials

### LINKED ISSUE #1225: feat: add tutorials for indexer replacement
https://blog.axelar.dev/how-to-deploy-an-application-with-the-same-address-cross-chain
https://blog.axelar.dev/cross-chain-governance-with-openzeppelin-governor-and-axelar
https://blog.axelar.dev/cross-chain-memecoins-with-axelar
https://blog.axelar.dev/multichain-rwa-lending-with-axelar-gmp
https://blog.axelar.dev/building-multichain-stablecoins-part-one
https://blog.axelar.dev/build-a-full-stack-interchain-application-with-nextjs-solidity-axelar

### PR DESCRIPTION
**Pull request by Olanetsoft**
Fixes #1225

### CODE REVIEW

File: src/layouts/navigation.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import { title } from "process";
```
Comment (StephenFluin) [inline src/layouts/navigation.ts:1]: I think this was an auto-import from your IDE that we don't want?

File: src/layouts/navigation.ts [hunk 2]
```diff
@@ -329,15 +330,31 @@ export const getNavigation = (section) => {
           ],
         },
         {
-          title: "Indexers",
+          title: "Tutorials",
```
Comment (StephenFluin) [inline src/layouts/navigation.ts:333]: You are removing the Indexers from the navigation, which I agree with (see #1203 ), but this should be a separate action as the content should be cleaned up as well.

**Review by StephenFluin — CHANGES_REQUESTED** [review]

**Comment by StephenFluin** [comment]
@Olanetsoft Did something happen to this PR? It looks like it includes a bunch of other changes now?

**Comment by Olanetsoft** [comment]
> Fixes #1225

I think I broke something while I was trying to rebase. I will close this now.

I have sent a new PR [here](https://github.com/axelarnetwork/axelar-docs/pull/1236)

**Pull request closed without merging**